Dockerfile

```diff
@@ -355,8 +331,8 @@ COPY --from=sysrepo /output/etc /etc/
 RUN export PYTHON3_VERSION=`python3 -c 'import sys; version=sys.version_info[:3]; print("python{0}.{1}".format(*version))'` && \
   echo "import site; site.addsitedir('/usr/local/lib/$PYTHON3_VERSION/site-packages')" \
     > /usr/local/lib/$PYTHON3_VERSION/dist-packages/use_site_packages.pth
-RUN export PYTHON2_VERSION=`python2 -c 'import sys; version=sys.version_info[:3]; print("python{0}.{1}".format(*version))'` && \
-  echo "import site; site.addsitedir('/usr/local/lib/$PYTHON2_VERSION/site-packages')" \
-    > /usr/local/lib/$PYTHON2_VERSION/dist-packages/use_site_packages.pth
+
+RUN ln -sf /usr/bin/python3 /usr/bin/python && \
+    ln -sf /usr/bin/pip3 /usr/bin/pip
```

Comment:
There is a package called [python-is-python3](https://packages.ubuntu.com/focal/python-is-python3) that might be a good idea to install. This will ensure that the commands using /usr/bin/python and Python scripts with #!/usr/bin/env python would continue to work.